src/custom/pages/Profile/index.tsx

```diff
@@ -84,6 +87,9 @@ export default function Profile() {
   const setSwapVCowStatus = useSetSwapVCowStatus()
   const swapVCowStatus = useSwapVCowStatus()
 
+  const [isLockedLoading, setIsLockedLoading] = useState(false)
+  const setLockedLoading = useCallback(() => setIsLockedLoading, [])
```

Comment:
It just doesn't work if I use `setIsLockedLoading` directly, because somehow it triggers the `loadingBalances` in lockedGNO card to end up true.

Reply:
You are right, this didn't do anything that is why it "worked". I've changed this by calling the hook in profile component and passing the data down to locked GNO so we have the loader in profile. Also I think it's better to get data there because otherwise the data for locked GNO doesn't get fetched until the spinner is removed and the locked GNO card is rendered which is not desired implementation